kilkari/src/main/java/org/motechproject/nms/kilkari/service/SubscriptionService.java

```diff
@@ -80,6 +80,19 @@ Subscription createSubscription(long callingNumber, Language language, Circle ci
     void activatePendingSubscriptionsUpTo(final DateTime upToDateTime);
 
     /**
+     * Activate subscriptions that are on hold
+     * @param maxAllowed max active subscriptions in the system
+     */
+    void toggleMctsSubscriptionCreation(long maxAllowed);
```

Comment:
The comment here doesn't match what the method does and the comment where it's implemented, or I'm crazy.